addons/mail/models/mail_mail.py

```diff
@@ -98,7 +101,12 @@ def process_email_queue(self, ids=None):
                                 messages are sent).
         """
         if not self.ids:
-            filters = [('state', '=', 'outgoing')]
+            import pdb; pdb.set_trace()
+            filters = ['&',
+                            ('state', '=', 'outgoing'),
```

Comment:
Ok for indenting; however here it is a bit too much ^^ . I would suggest to align everything vertically.